client/app/_components/layout/footer/Footer.tsx

```diff
@@ -19,22 +19,22 @@ const Footer = ({ hasNavigation = true }: FooterProps) => {
           aria-label='Visit home page'
           className='flex items-center justify-center rounded-full text-primary transition-transform duration-300 hover:scale-110'
         >
-          <BiHomeAlt2 className='h-7 w-7' />
+          <FiHome className='h-7 w-7' />
         </Link>
         <Link
           href={'/tasks'}
           aria-label='Visit tasks page'
           className='relative flex -translate-y-8 items-center justify-center'
         >
-          <BiCalendarAlt className='z-10 h-12 w-12 rounded-full bg-primary p-2 shadow-md shadow-primary transition-transform duration-300 hover:scale-110' />
+          <FiCalendar className='z-10 h-12 w-12 rounded-full bg-primary p-2 shadow-md shadow-primary transition-transform duration-300 hover:scale-110' />
```

Comment:
The bottom is a little bit chipped! You need to fix padding like `px-2 py-1`

Reply:
Thanks! fixed it. 
609d122e9fb262ea5f8fbcfa3d7e42f01e8688d9